inf-terraform-aws/files/Makefile

```diff
@@ -156,32 +160,4 @@ __check_defined = \
 	$(if $(value $1),, \
 		$(error Undefined $1$(if $2, ($2))$(if $(value @), \
 			required by target `$@')))
-
-# Execute tests in included blueprints.
```

Comment:
is this removal part of this PR scope? why is theses tests removed here? just curious

Reply:
code is not in use